src/renderers/webgl/WebGLUtils.d.ts

```diff
@@ -0,0 +1,5 @@
+export class WebGLUtils {
+	constructor(gl: CanvasRenderingContext2D | WebGLRenderingContext, extensions: any, capabilities: any);
```

Comment:
I think it's wrong to use `CanvasRenderingContext2D` which represent a 2D drawing context. You always create a WebGL or WebGL2 context. Hence it should be 

`gl: WebGLRenderingContext | WebGL2RenderingContext`

Reply:
BTW: This is also wrong in `WebGLAnimation.d.ts` and `WebGLAttributes.d.ts`.

Reply:
Solved via #16458 😊